### DISCUSSION TITLE
antsMotionCorrStats 3d or 4d mask representing what

### DISCUSSION THREAD

**Discussion by scott-love** [ANTsX/ANTs#1730] [Q&A] [answered]
Hi all, 

I am trying to understand whether the required mask for antsMotionCorrStats should have 3 or 4 dimensions?
I think it should be 4D because the position of the object (e.g., brain) that the mask represents could be different at each time point, hence the need to actually do motion correction. 

A related question as to what exactly should the mask represent. Should it be a mask of **all non-zero voxels** (at each time point if it should be 4d)? When writing this question I just had a doubt as to whether it should actually be a fixed (and not a moving) image mask? I guess I don't totally understand why a mask and not the actual image intensity data is required. 

Any insights would be greatly appreciated.
Scott

**Comment by cookpa** [comment, marked as answer]
`antsMotionCorrStats` is run after motion correction with `antsMotionCorr`, so the volumes should all be aligned. The 3D mask is used to restrict metrics like voxelwise framewise displacement to voxels in the brain. A good way to make the mask is to skull-strip the average volume after motion correction.

The program will use the mask and the motion correction transforms to compute how much motion there was inside brain voxels, by summing up the displacement of each voxel in the brain during correction.

**Reply by scott-love** [reply]
That helps, thanks! I got stuck thinking the mask needed to be in the original space from before motion correction.
